hack/list-swagger-objects.py

```diff
@@ -0,0 +1,13 @@
+import json;
```

Comment:
Upstream does not have this code. Upstream `grep`s through `register.go` to find everything that has an `IsAnAPIObject` method. This is not the same, as the items that Swagger picks up.